### PR TITLE
Configuration and theme updates

### PR DESCRIPTION
**Pull request by deniss-mc**
**Jira issue:**
https://glia.atlassian.net/browse/MUIC-302

### CODE REVIEW

File: widgetssdk/src/main/java/com/glia/widgets/UiTheme.java [hunk 1]
```diff
@@ -208,6 +208,12 @@
     private @DrawableRes
     final Integer iconPlaceholder;
 
+    /**
+     * The icon resId used when visitor is put on hold by the operator
+     */
+    private @DrawableRes
+    final Integer iconOnHold;
+
```
Comment (jakob-mc) [inline widgetssdk/src/main/java/com/glia/widgets/UiTheme.java:216]: pls add empty line for other properties as well

File: widgetssdk/src/main/java/com/glia/widgets/view/configuration/ChatHeadConfiguration.java [hunk 2]
```diff
@@ -11,6 +11,9 @@
     private final Integer badgeTextColor;
     private final Integer backgroundColorRes;
 
+    private final Integer onHoldIcon;
```
Comment (jakob-mc) [inline widgetssdk/src/main/java/com/glia/widgets/view/configuration/ChatHeadConfiguration.java:14]: in the theme it's called `iconOnHold`, here it's a bit different. would be nice to keep it consistent throughout the codebase.

File: widgetssdk/src/main/java/com/glia/widgets/view/configuration/ChatHeadConfiguration.java [hunk 3]
```diff
@@ -187,6 +226,16 @@ public Builder backgroundColorRes(Integer backgroundColorRes) {
             return this;
         }
 
+        public Builder onHoldIcon(Integer onHoldIcon) {
```
Comment (jakob-mc) [inline widgetssdk/src/main/java/com/glia/widgets/view/configuration/ChatHeadConfiguration.java:229]: just a question, why Integer instead int?

File: widgetssdk/src/main/res/values/attrs.xml [hunk 4]
```diff
@@ -140,6 +143,8 @@
     <attr name="gliaIconCallMinimize" format="reference" />
     <!-- The icon resId used in a chat head if an operator's profile image is absent -->
     <attr name="gliaIconPlaceholder" format="reference" />
+    <!-- The icon resId used in views to indicate operator put visitor on hold -->
```
Comment (jakob-mc) [inline widgetssdk/src/main/res/values/attrs.xml:146]: can it be same wording as above to make it more clear?

**Review by jakob-mc — CHANGES_REQUESTED** [review]

File: widgetssdk/src/main/java/com/glia/widgets/view/configuration/ChatHeadConfiguration.java (hunk 3 above)
Comment (deniss-mc) [reply widgetssdk/src/main/java/com/glia/widgets/view/configuration/ChatHeadConfiguration.java:229]: Runtime value could be null - if its null then we dont override the buildtime value

File: widgetssdk/src/main/res/values/attrs.xml (hunk 4 above)
Comment (deniss-mc) [reply widgetssdk/src/main/res/values/attrs.xml:146]: Add suggestion

File: widgetssdk/src/main/java/com/glia/widgets/view/configuration/ChatHeadConfiguration.java (hunk 3 above)
Comment (jakob-mc) [reply widgetssdk/src/main/java/com/glia/widgets/view/configuration/ChatHeadConfiguration.java]: clear

File: widgetssdk/src/main/res/values/attrs.xml (hunk 4 above)
Comment (jakob-mc) [reply widgetssdk/src/main/res/values/attrs.xml:146]: `<!-- The icon resId used when visitor is put on hold by operator -->`
vs
`<!-- The icon resId used in views to indicate operator put visitor on hold -->`

**Review by jakob-mc — APPROVED** [review]

**Pull request merged**